cmd/argocd/commands/app.go

```diff
@@ -1076,9 +1082,17 @@ func printApplicationTable(apps []argoappv1.Application, output *string) {
 	}
 	_, _ = fmt.Fprintf(w, fmtStr, headers...)
 	for _, app := range apps {
+
+		var appServer string
```

Comment:
Could you please move this part to private function that will return a string ( server ) and reuse it in code above , so it will reduce duplication 
fmt.Printf(printOpFmtStr, "Server:", getServer())

Reply:
Thanks, made changes